src/SIGV4ConnectionConfig.test.tsx

```diff
@@ -0,0 +1,97 @@
+import { DataSourcePluginOptionsEditorProps } from '@grafana/data';
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { SIGV4ConnectionConfig } from 'SIGV4ConnectionConfig';
+import { AwsAuthType } from 'types';
+
+const resetWindow = () => {
+  (window as any).grafanaBootData = {
```

Comment:
I suspect in this case it does not matter however a pattern I have seen done for this kind of thing before is to do something like:
```
const old = window.grafanaBootData
// everything you have already
afterEach(() => window.grafanaBootData = old)
```

I think otherwise there might be a possibility that in some other test you might be testing that window.grafanaBootData gets attached to window properly and it will fail or give you a false positive because it's carrying over the global window object between test files. 

Kind of nitpicky but maybe worth it? I'm not terribly familiar with this property so I don't really know.

Oh also just remembering that I think jest might have some tooling for this: https://stackoverflow.com/questions/41885841/how-can-i-mock-the-javascript-window-object-using-jest